static/app/components/searchQueryBuilder/types.tsx

```diff
@@ -25,3 +25,21 @@ export type CallbackSearchState = {
   parsedQuery: ParseResult | null;
   queryIsValid: boolean;
 };
+
+/**
+ * This is a list of wildcard operators that are used in the search query builder.
+ * These are only present on the frontend. This is because we utilize the underlying
+ * '*' character rather than introducing new operators on the backend.
+ */
+export enum WildcardOperators {
+  CONTAINS = 'contains',
+  DOES_NOT_CONTAIN = 'does not contain',
+  STARTS_WITH = 'starts with',
+  ENDS_WITH = 'ends with',
+}
+
+export function isTermOperator(op: FETermOperators): op is TermOperator {
+  return typeof op === 'string' && Object.values(TermOperator).includes(op as any);
+}
+
+export type FETermOperators = TermOperator | WildcardOperators;
```

Comment:
nit: would prefer something like `SearchQueryBuilderOperators` insetad, but naming is hard 😅

Reply:
Will push a quick change!